torch/fx/experimental/proxy_tensor.py

```diff
@@ -710,6 +696,13 @@ def get_torch_dispatch_modes():
     return torch.utils._python_dispatch._get_current_dispatch_mode_stack()
 
 
+def get_innermost_proxy_mode():
+    for m in reversed(torch.utils._python_dispatch._get_current_dispatch_mode_stack()):
```

Comment:
curiosity question: under what situation can we have multiple proxy modes on the stack at once?

Reply:
get_isolated_graphmodule